### PR TITLE
Sans barriers

### PR DESCRIPTION
**Pull request by commuterjoy**
- [x] Remove article code from `next-article`
- [x] Move barriers to `next-product-selector`
- [x] Think of less bad name than product selector - Eg, ft.com/packages
- [x] Deprecate `next-barriers` middleware
- [x] Route barriers to product selector in fastly
- [x] Carry the article title through to some barriers
- [x] Ensure `vary` headers & `stale-if-error` are retained in article and product-selector
- [x] Scale up product selector app to a few more dynos

### CODE REVIEW

**Comment by i-like-robots** [comment]
@commuterjoy Want me to pick this up and push this de-barrier-ification through?

**Comment by commuterjoy** [comment]
Two more things - 

- [ ] Ship https://github.com/Financial-Times/next-cdn/pull/15
- [ ] Modify https://github.com/Financial-Times/next-product-selector to render corporate barriers

File: server/app.js [hunk 1]
```diff
@@ -25,19 +24,12 @@ app.post('^/preview$', require('./controllers/preview'));
 
 const uuid = '[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}';
 
-// COMPLEX: Put access middleware before barrier middleware so that access can be cached by membership
-app.use(`^/content/:id(${uuid})$`, require('./controllers/access'));
```
Comment (commuterjoy) [inline server/app.js]: @i-like-robots is this deletion a mistake?

Comment (i-like-robots) [inline server/app.js]: Previous commit removed this too https://github.com/Financial-Times/next-article/commit/afa6f7227871e3984f79025ea76d45b7b722c1ba

Comment (i-like-robots) [inline server/app.js]: It smells like a mistake because the controller itself and related test etc. haven't been removed.

Comment (commuterjoy) [inline server/app.js]: so if the article app is still used by access then I should keep this line

Comment (i-like-robots) [inline server/app.js]: I would think so!

**Pull request merged**